### ISSUE TITLE
2025-04-21, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#147]
# Today's Arxiv Papers

(1) [Does Reinforcement Learning Really Incentivize Reasoning Capacity in LLMs Beyond the Base Model?](http://arxiv.org/abs/2504.13837v1), 2025-04-18

(2) [Science Hierarchography: Hierarchical Organization of Science Literature](http://arxiv.org/abs/2504.13834v1), 2025-04-18

(3) [Generative AI Act II: Test Time Scaling Drives Cognition Engineering](http://arxiv.org/abs/2504.13828v1), 2025-04-18

(4) [Feature Alignment and Representation Transfer in Knowledge Distillation for Large Language Models](http://arxiv.org/abs/2504.13825v1), 2025-04-18

(5) [Not All Rollouts are Useful: Down-Sampling Rollouts in LLM Reinforcement Learning](http://arxiv.org/abs/2504.13818v1), 2025-04-18

(6) [Analyzing LLMs' Knowledge Boundary Cognition Across Languages Through the Lens of Internal Representations](http://arxiv.org/abs/2504.13816v1), 2025-04-18

(7) [BadApex: Backdoor Attack Based on Adaptive Optimization Mechanism of Black-box Large Language Models](http://arxiv.org/abs/2504.13775v1), 2025-04-18

(8) [DP2Unlearning: An Efficient and Guaranteed Unlearning Framework for LLMs](http://dx.doi.org/10.2139/ssrn.5217160), 2025-04-18

(9) [Detecting Malicious Source Code in PyPI Packages with LLMs: Does RAG Come in Handy?](http://arxiv.org/abs/2504.13769v1), 2025-04-18

(10) [Scaling sparse feature circuit finding for in-context learning](http://arxiv.org/abs/2504.13756v1), 2025-04-18

(11) [Controlled Territory and Conflict Tracking (CONTACT): (Geo-)Mapping Occupied Territory from Open Source Intelligence](http://arxiv.org/abs/2504.13730v1), 2025-04-18

(12) [OpenDeception: Benchmarking and Investigating AI Deceptive Behaviors via Open-ended Interaction Simulation](http://arxiv.org/abs/2504.13707v1), 2025-04-18

(13) [Exploring Multimodal Prompt for Visualization Authoring with Large Language Models](http://arxiv.org/abs/2504.13700v1), 2025-04-18

(14) [Intelligent Interaction Strategies for Context-Aware Cognitive Augmentation](http://arxiv.org/abs/2504.13684v1), 2025-04-18

(15) [Revisiting Uncertainty Quantification Evaluation in Language Models: Spurious Interactions with Response Length Bias Results](http://arxiv.org/abs/2504.13677v1), 2025-04-18

(16) [Large Language Models Will Change The Way Children Think About Technology And Impact Every Interaction Paradigm](http://arxiv.org/abs/2504.13667v1), 2025-04-18

(17) [Do Prompt Patterns Affect Code Quality? A First Empirical Assessment of ChatGPT-Generated Code](http://arxiv.org/abs/2504.13656v1), 2025-04-18

(18) [Exploring the Potential for Large Language Models to Demonstrate Rational Probabilistic Beliefs](http://arxiv.org/abs/2504.13644v1), 2025-04-18

(19) [Remedy: Learning Machine Translation Evaluation from Human Preferences with Reward Modeling](http://arxiv.org/abs/2504.13630v1), 2025-04-18

(20) [Divergent LLM Adoption and Heterogeneous Convergence Paths in Research Writing](http://arxiv.org/abs/2504.13629v1), 2025-04-18

(21) [PV-VLM: A Multimodal Vision-Language Approach Incorporating Sky Images for Intra-Hour Photovoltaic Power Forecasting](http://arxiv.org/abs/2504.13624v1), 2025-04-18

(22) [Visual Intention Grounding for Egocentric Assistants](http://arxiv.org/abs/2504.13621v1), 2025-04-18

(23) [Compile Scene Graphs with Reinforcement Learning](http://arxiv.org/abs/2504.13617v1), 2025-04-18

(24) [Long-context Non-factoid Question Answering in Indic Languages](http://arxiv.org/abs/2504.13615v1), 2025-04-18

(25) [Continual Pre-Training is (not) What You Need in Domain Adaption](http://arxiv.org/abs/2504.13603v1), 2025-04-18

(26) [Improving Generalization in Intent Detection: GRPO with Reward-Based Curriculum Sampling](http://arxiv.org/abs/2504.13592v1), 2025-04-18

(27) [Towards End-to-End Network Intent Management with Large Language Models](http://arxiv.org/abs/2504.13589v1), 2025-04-18

(28) [RAG Without the Lag: Interactive Debugging for Retrieval-Augmented Generation Pipelines](http://arxiv.org/abs/2504.13587v1), 2025-04-18

(29) [Contextualizing Spotify's Audiobook List Recommendations with Descriptive Shelves](http://arxiv.org/abs/2504.13572v1), 2025-04-18

(30) [DETAM: Defending LLMs Against Jailbreak Attacks via Targeted Attention Modification](http://arxiv.org/abs/2504.13562v1), 2025-04-18

(31) [Zero-Shot Industrial Anomaly Segmentation with Image-Aware Prompt Generation](http://arxiv.org/abs/2504.13560v1), 2025-04-18

(32) [Integrating LLMs for Grading and Appeal Resolution in Computer Science Education](http://arxiv.org/abs/2504.13557v1), 2025-04-18

(33) [CoT-RAG: Integrating Chain of Thought and Retrieval-Augmented Generation to Enhance Reasoning in Large Language Models](http://arxiv.org/abs/2504.13534v1), 2025-04-18

(34) [Large Language Models for Validating Network Protocol Parsers](http://arxiv.org/abs/2504.13515v1), 2025-04-18

(35) [Prejudge-Before-Think: Enhancing Large Language Models at Test-Time by Process Prejudge Reasoning](http://arxiv.org/abs/2504.13500v1), 2025-04-18

(36) [Early Timestep Zero-Shot Candidate Selection for Instruction-Guided Image Editing](http://arxiv.org/abs/2504.13490v1), 2025-04-18

(37) [LLM Sensitivity Evaluation Framework for Clinical Diagnosis](http://arxiv.org/abs/2504.13475v1), 2025-04-18

(38) [Everything You Wanted to Know About LLM-based Vulnerability Detection But Were Afraid to Ask](http://arxiv.org/abs/2504.13474v1), 2025-04-18

(39) [CodeVisionary: An Agent-based Framework for Evaluating Large Language Models in Code Generation](http://arxiv.org/abs/2504.13472v1), 2025-04-18

(40) [From Large to Super-Tiny: End-to-End Optimization for Cost-Efficient LLMs](http://arxiv.org/abs/2504.13471v1), 2025-04-18

(41) [Chain-of-Thought Textual Reasoning for Few-shot Temporal Action Localization](http://arxiv.org/abs/2504.13460v1), 2025-04-18

(42) [Balancing Engagement and Polarization: Multi-Objective Alignment of News Content Using LLMs](http://arxiv.org/abs/2504.13444v1), 2025-04-18

(43) [Trust, but verify](http://arxiv.org/abs/2504.13443v1), 2025-04-18

(44) [Secure Multifaceted-RAG for Enterprise: Hybrid Knowledge Retrieval with Security Filtering](http://arxiv.org/abs/2504.13425v1), 2025-04-18

(45) [STAMP Your Content: Proving Dataset Membership via Watermarked Rephrasings](http://arxiv.org/abs/2504.13416v1), 2025-04-18

(46) [Towards a Multi-Agent Vision-Language System for Zero-Shot Novel Hazardous Object Detection for Autonomous Driving Safety](http://arxiv.org/abs/2504.13399v1), 2025-04-18

(47) [Using LLMs for Library Migration](http://arxiv.org/abs/2504.13272v1), 2025-04-17

(48) [Causal-Copilot: An Autonomous Causal Analysis Agent](http://arxiv.org/abs/2504.13263v1), 2025-04-17

(49) [CPG-EVAL: A Multi-Tiered Benchmark for Evaluating the Chinese Pedagogical Grammar Competence of Large Language Models](http://arxiv.org/abs/2504.13261v1), 2025-04-17

(50) [Aligning Constraint Generation with Design Intent in Parametric CAD](http://arxiv.org/abs/2504.13178v1), 2025-04-17

(51) [Sleep-time Compute: Beyond Inference Scaling at Test-time](http://arxiv.org/abs/2504.13171v1), 2025-04-17

(52) [Exploring Expert Failures Improves LLM Agent Tuning](http://arxiv.org/abs/2504.13145v1), 2025-04-17

(53) [Syntactic and Semantic Control of Large Language Models via Sequential Monte Carlo](http://arxiv.org/abs/2504.13139v1), 2025-04-17

(54) [Energy-Based Reward Models for Robust Language Model Alignment](http://arxiv.org/abs/2504.13134v1), 2025-04-17

(55) [LLMs Meet Finance: Fine-Tuning Foundation Models for the Open FinLLM Leaderboard](http://arxiv.org/abs/2504.13125v1), 2025-04-17

(56) [Low-hallucination Synthetic Captions for Large-Scale Vision-Language Model Pre-training](http://arxiv.org/abs/2504.13123v1), 2025-04-17

(57) [VistaDPO: Video Hierarchical Spatial-Temporal Direct Preference Optimization for Large Video Models](http://arxiv.org/abs/2504.13122v1), 2025-04-17

(58) [Uncertainty-Aware Trajectory Prediction via Rule-Regularized Heteroscedastic Deep Classification](http://arxiv.org/abs/2504.13111v1), 2025-04-17

(59) [EventVAD: Training-Free Event-Aware Video Anomaly Detection](http://arxiv.org/abs/2504.13092v1), 2025-04-17

(60) [Retrieval-Augmented Generation with Conflicting Evidence](http://arxiv.org/abs/2504.13079v1), 2025-04-17

(61) [ImPart: Importance-Aware Delta-Sparsification for Improved Model Compression and Merging in LLMs](http://arxiv.org/abs/2504.13237v1), 2025-04-17

(62) [SkyReels-V2: Infinite-length Film Generative Model](http://arxiv.org/abs/2504.13074v2), 2025-04-17

(63) [Early Accessibility: Automating Alt-Text Generation for UI Icons During App Development](http://arxiv.org/abs/2504.13069v1), 2025-04-17

(64) [Accuracy is Not Agreement: Expert-Aligned Evaluation of Crash Narrative Classification Models](http://arxiv.org/abs/2504.13068v1), 2025-04-17

(65) [NNTile: a machine learning framework capable of training extremely large GPT language models on a single node](http://arxiv.org/abs/2504.13236v1), 2025-04-17

(66) [RoboTwin: Dual-Arm Robot Benchmark with Generative Digital Twins](http://arxiv.org/abs/2504.13059v1), 2025-04-17

(67) [Aspect-Based Summarization with Self-Aspect Retrieval Enhanced Generation](http://arxiv.org/abs/2504.13054v1), 2025-04-17

(68) [GraphAttack: Exploiting Representational Blindspots in LLM Safety Mechanisms](http://arxiv.org/abs/2504.13052v1), 2025-04-17

(69) [Deep literature reviews: an application of fine-tuned language models to migration research](http://arxiv.org/abs/2504.13685v1), 2025-04-17

(70) [How Large Language Models Are Changing MOOC Essay Answers: A Comparison of Pre- and Post-LLM Responses](http://arxiv.org/abs/2504.13038v1), 2025-04-17

(71) [InstructRAG: Leveraging Retrieval-Augmented Generation on Instruction Graphs for LLM-Based Task Planning](http://arxiv.org/abs/2504.13032v1), 2025-04-17

(72) [ChatEXAONEPath: An Expert-level Multimodal Large Language Model for Histopathology Using Whole Slide Images](http://arxiv.org/abs/2504.13023v1), 2025-04-17

(73) [Auto-FEDUS: Autoregressive Generative Modeling of Doppler Ultrasound Signals from Fetal Electrocardiograms](http://arxiv.org/abs/2504.13233v1), 2025-04-17

(74) [SHA256 at SemEval-2025 Task 4: Selective Amnesia -- Constrained Unlearning for Large Language Models via Knowledge Isolation](http://arxiv.org/abs/2504.12996v1), 2025-04-17

(75) [Chain-of-Thought Prompting for Out-of-Distribution Samples: A Latent-Variable Study](http://arxiv.org/abs/2504.12991v1), 2025-04-17

(76) [A Virtual Machine for Arbitrary Low-Precision GPGPU Computation in LLM Serving](http://arxiv.org/abs/2504.12984v1), 2025-04-17

(77) [Accommodate Knowledge Conflicts in Retrieval-augmented LLMs: Towards Reliable Response Generation in the Wild](http://arxiv.org/abs/2504.12982v1), 2025-04-17

(78) [Sparks of Science: Hypothesis Generation Using Structured Paper Data](http://arxiv.org/abs/2504.12976v1), 2025-04-17

(79) [Estimating Optimal Context Length for Hybrid Retrieval-augmented Multi-document Summarization](http://arxiv.org/abs/2504.12972v1), 2025-04-17

(80) [QLLM: Do We Really Need a Mixing Network for Credit Assignment in Multi-Agent Reinforcement Learning?](http://arxiv.org/abs/2504.12961v1), 2025-04-17

(81) [Are Retrials All You Need? Enhancing Large Language Model Reasoning Without Verbalized Feedback](http://arxiv.org/abs/2504.12951v1), 2025-04-17

(82) [Customizing Emotional Support: How Do Individuals Construct and Interact With LLM-Powered Chatbots](http://arxiv.org/abs/2504.12943v2), 2025-04-17

(83) [Explainable AI in Usable Privacy and Security: Challenges and Opportunities](http://arxiv.org/abs/2504.12931v1), 2025-04-17

(84) [DIDS: Domain Impact-aware Data Sampling for Large Language Model Training](http://arxiv.org/abs/2504.13227v1), 2025-04-17

(85) [ConExion: Concept Extraction with Large Language Models](http://arxiv.org/abs/2504.12915v1), 2025-04-17

(86) [MAIN: Mutual Alignment Is Necessary for instruction tuning](http://arxiv.org/abs/2504.12913v1), 2025-04-17

(87) [Benchmarking Multi-National Value Alignment for Large Language Models](http://arxiv.org/abs/2504.12911v1), 2025-04-17

(88) [Information Gain-Guided Causal Intervention for Autonomous Debiasing Large Language Models](http://arxiv.org/abs/2504.12898v1), 2025-04-17

(89) [EmoVoice: LLM-based Emotional Text-To-Speech Model with Freestyle Text Prompting](http://arxiv.org/abs/2504.12867v2), 2025-04-17

(90) [DashChat: Interactive Authoring of Industrial Dashboard Design Prototypes through Conversation with LLM-Powered Agents](http://arxiv.org/abs/2504.12865v1), 2025-04-17

(91) [Can LLMs reason over extended multilingual contexts? Towards long-context evaluation beyond retrieval and haystacks](http://arxiv.org/abs/2504.12845v1), 2025-04-17

(92) [Assesing LLMs in Art Contexts: Critique Generation and Theory of Mind Evaluation](http://arxiv.org/abs/2504.12805v1), 2025-04-17

(93) [EarthGPT-X: Enabling MLLMs to Flexibly and Comprehensively Understand Multi-Source Remote Sensing Imagery](http://arxiv.org/abs/2504.12795v1), 2025-04-17

(94) [Enhancing the Geometric Problem-Solving Ability of Multimodal LLMs via Symbolic-Neural Integration](http://arxiv.org/abs/2504.12773v1), 2025-04-17

(95) [GraphOmni: A Comprehensive and Extendable Benchmark Framework for Large Language Models on Graph-theoretic Tasks](http://arxiv.org/abs/2504.12764v1), 2025-04-17

(96) [Trajectory Adaptation using Large Language Models](http://arxiv.org/abs/2504.12755v1), 2025-04-17

(97) [Chinese-Vicuna: A Chinese Instruction-following Llama-based Model](http://arxiv.org/abs/2504.12737v1), 2025-04-17

(98) [Pandora: A Code-Driven Large Language Model Agent for Unified Reasoning Across Diverse Structured Knowledge](http://arxiv.org/abs/2504.12734v1), 2025-04-17

(99) [Validating LLM-Generated Relevance Labels for Educational Resource Search](http://arxiv.org/abs/2504.12732v1), 2025-04-17

(100) [SimUSER: Simulating User Behavior with Large Language Models for Recommender System Evaluation](http://arxiv.org/abs/2504.12722v1), 2025-04-17
